lib/dotcom/transit_near_me.ex

```diff
@@ -197,6 +194,10 @@ defmodule Dotcom.TransitNearMe do
 
   def format_prediction_time(_, _, :bus, _), do: ["arriving"]
 
+  def format_prediction_time(%DateTime{} = time, _now, _, _) do
```

Comment:
If we ever add support for ferry predictions, this'll have us covered (and format them the same as commuter rail ones, which is a pattern we follow elsewhere when showing schedules)!